recipe/meta.yaml

```diff
@@ -16,13 +16,15 @@ build:
 
 requirements:
   host:
-    - python >=3.7
+    - python >=3.8
     - pip
+    - flit_core >=3.2,<4
```

Comment:
```suggestion
    - flit-core >=3.2,<4
```

Reply:
Oh my, thank you :)
This had me perplexed, I was going to check again in the morning hehe